src/main/java/io/vavr/control/Either.java

```diff
@@ -193,6 +193,18 @@ private Either() {
     /**
      * Maps either the left or the right side of this disjunction.
      *
+     * <pre>{@code
+     *Either<?, AtomicInteger> success = Either.right(new AtomicInteger(42));
+     *
+     * //prints "Right(42)"
```

Comment:
Please add one space after `//` as we do it across the whole code base for reasons of consistency and readability.

Reply:
Fixed, also in previous change.